src/utils/validateTokenPair.test.js

```diff
@@ -0,0 +1,16 @@
+/* global shallow describe it expect */
+/* eslint-disable import/first */
+import React from 'react';
+import { validateTokenPair } from './validateTokenPair';
+import { generateTradingPairs } from './generateTradingPairs';
+import { BASE_TOKENS, QUOTE_TOKENS } from '../constant';
+
+describe('(Util) validateTokenPair', () => {
```

Comment:
Actually you have at  4 scenarios in here.

1. If you haven't provided baseToken.
2. If you haven't provided quoteToken.
3. If you provided them but there is no existing pair in the list
4. If everything is fine and there is pair.